tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -75,4 +75,11 @@ build_rootfs() {
     # Reduce image size and memory footprint
     # removing not needed files and directories.
     chroot $ROOTFS_DIR rm -rf /usr/share/{bash-completion,bug,doc,info,lintian,locale,man,menu,misc,pixmaps,terminfo,zoneinfo,zsh}
+
+ 	if [ "${AA_KBC}" == "eaa_kbc" ]; then
+		echo 'deb [arch=amd64] http://mirrors.openanolis.cn/inclavare-containers/ubuntu20.04 bionic main' | chroot $ROOTFS_DIR tee /etc/apt/sources.list.d/inclavare-containers.list
+		wget -qO - http://mirrors.openanolis.cn/inclavare-containers/ubuntu20.04/DEB-GPG-KEY.key  | chroot $ROOTFS_DIR apt-key add -
+		chroot $ROOTFS_DIR apt-get update
+		chroot $ROOTFS_DIR apt-get install -y rats-tls
```

Comment:
I think this should be a single chroot.

Reply:
I tried command "chroot $ROOTFS_DIR apt-get update && apt-get install -y rats-tls" and found it unworkable.